ios/Classes/readers/configurations/NetworkConfigurationReader.swift

```diff
@@ -15,14 +15,22 @@ import SnowplowTracker
 struct NetworkConfigurationReader: Decodable {
     let endpoint: String
     let method: String?
+    let customPostPath: String?
 }
 
 extension NetworkConfigurationReader {
     func toConfiguration() -> NetworkConfiguration {
-        if let m = method {
-            return NetworkConfiguration(endpoint: endpoint, method: m == "get" ? .get : .post)
-        } else {
-            return NetworkConfiguration(endpoint: endpoint, method: .post)
-        }
+    let networkConfig: NetworkConfiguration
+
+    if let m = method {
+        networkConfig = NetworkConfiguration(endpoint: endpoint, method: m == "get" ? .get : .post)
+    } else {
+        networkConfig = NetworkConfiguration(endpoint: endpoint, method: .post)
+    }
+
+    if let c = customPostPath {
+        networkConfig.customPostPath("/\(c)")
```

Comment:
It didn't add its own slash. I checked the Simulator logs using Console and found that it was using the path "https://<numbers>.ngrok.iocom.custom", which wasn't valid so no events were sent.

Reply:
Oh right, that must be because the behaviour changed a bit in version 4 where we fixed a bug related to it: https://github.com/snowplow/snowplow-objc-tracker/pull/703

Reply:
Maybe lets first upgrade the mobile trackers to v4 (issue #17) and then update this?

Reply:
To be honest, v4 should accept both with and without the leading slash so we could also leave it like this, just feels a bit cleaner to not modify the input.

Reply:
Good plan, we can update this later.